### DISCUSSION TITLE
buildx docker CLI plugin missing in all Renovate docker images

### DISCUSSION THREAD

**Discussion by mtardy** [renovatebot/renovate#22435] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.98.4

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

In the renovate Docker images, the Docker CLI is installed but buildx CLI plugin is missing. Do you think it would be possible to include it? It's not present in the "normal" and "slim" images.

### What you tried so far.

To allow commands to work, I had to add a manual install pre-command:
```shell
mkdir -p $HOME/.docker/cli-plugins && curl -L https://github.com/docker/buildx/releases/download/v0.10.5/buildx-v0.10.5.linux-amd64 -o $HOME/.docker/cli-plugins/docker-buildx && chmod +x $HOME/.docker/cli-plugins/docker-buildx
```
As [my first command](https://github.com/mtardy/tetragon/blob/main/.github/renovate.json5#L73) in my `postUpgradeTasks`'s commands.

This question is related to [this other question](https://github.com/renovatebot/renovate/discussions/22379).

**Comment by viceice** [comment, marked as answer]
it's intended to not install, because renovate doesn't need it. there is also no plans to change that.
